includes/admin/class-sensei-onboarding.php

```diff
@@ -19,6 +19,7 @@
  * @since   3.1.0
  */
 class Sensei_Onboarding {
+	const SUGGEST_ONBOARDING_OPTION = 'sensei_suggest_onboarding';
```

Comment:
It looks like for newer options we might be using `-` characters instead of `_`. But I'm not sure whether we are doing that consistently. @jom any thoughts?

Reply:
I think `_` actually seems to be more standard for options. Enrolment related options used `_`.

Reply:
Yeah. There is a mix of both :(
But considering the latest are using `_`, I think we could get `_` as standard. Make sense?

Reply:
Works for me. Let's use `_` moving forward.